plugin.py

```diff
@@ -17,6 +21,25 @@ class LspTailwindcssPlugin(NpmClientHandler):
         server_directory, 'extension', 'dist', 'server', 'index.js'
     )
 
+    @classmethod
+    def is_allowed_to_start(
+        cls,
+        _window: sublime.Window,
+        _initiating_view: Optional[sublime.View] = None,
+        workspace_folders: Optional[List[WorkspaceFolder]] = None,
+        _configuration: Optional[ClientConfig] = None
+    ) -> Optional[str]:
+        if not workspace_folders:
+            return "{} - requires a folder to start.".format(cls.package_name)
+        path = workspace_folders[0].path
+        tailwind_config_file_path = os.path.join(path, 'tailwind.config.js')
+        if not os.path.exists(tailwind_config_file_path):
+            return "{} - no tailwind.config.js present in {}".format(cls.package_name, path)
+        is_tailwind_dependecy_installed = os.path.join(path, 'node_modules', 'tailwindcss')
+        if not os.path.exists(is_tailwind_dependecy_installed):
+            return "{} - No tailwindcss package found in node_modules for path {}. Make sure to install the tailwindcss npm package.".format(cls.package_name, path)
```

Comment:
Checking for the package in node_modules would not work with yarn pnp. I don't know if the server supports that case but if it does then you'll prevent it from working here.

Reply:
6d29cba

I removed the dependency check. 
Don't want to prevent the plugin from not starting for yarn pnp users.
But also don't want to complicate the logic have multiple if statements for checking if the dependency is installed.

If some users stumble upon this issue I would revisit this and add the necessary checks,
but for now I think just checking the presence of tailwind.config.js file and the description in the README is sufficient to guide the user to make this plugin work.